app/uk/gov/hmrc/vulnerabilities/persistence/VulnerabilitySummariesRepository.scala

```diff
@@ -43,6 +45,7 @@ class VulnerabilitySummariesRepository @Inject()(
   domainFormat   = VulnerabilitySummary.mongoFormat,
   indexes        = Seq(
     IndexModel(Indexes.ascending("occurrences.service"), IndexOptions().name("occurrences.service").background(true)),
+    IndexModel(Indexes.ascending("occurrences.componentPathInSlug"), IndexOptions().name("occurrences.componentPathInSlug").background(true)),
```

Comment:
You're not using this index currently - you're doing a regex match (in a projection) - I'd be tempted to only add if and when we need it.